client/app/components/breadcrumbs.hbs

```diff
@@ -0,0 +1,10 @@
+{{!-- TODO: Make this hardcoded breadcrumb trail dynamic--}}
+
+<nav aria-label="You are here:" role="navigation">
+  <ul class="breadcrumbs">
+    <li><a href="/projects.html">My Projects</a></li>
+    <li class="disabled">123 Project Name</li>
+    <li class="disabled">Pre-Application Statement</li>
+    <li><span class="show-for-sr">Current: </span> Edit </li>
+  </ul>
+</nav>
```

Comment:
If you created a component-class (`ember g component-class`) and inject the router service, i think you can get access to some of the info you'd need for this.

Reply:
Thanks. I'll open another issue to make the breadcrumbs dynamic. Wanna get this one in sooner.